### DISCUSSION TITLE
Turn json into page

### DISCUSSION THREAD

**Discussion by seananth** [prevwong/craft.js#290] [Q&A] [answered]
Hey,

Lets say for example I want to build a basic form builder, there is an input and a submit button. After we serialized the component to json like shown in the docs, how would you actually build a page from it?

I found this issue  #42  but it doesn't really fit as that is only markup.

How would you go about it?

**Comment by ankri** [comment, marked as answer]
You can take the JSON and render inside an `<Editor />` that is disabled. Like in the [basic example](https://craft.js.org/examples/basic/).

In my project I use a view with an `enabled` `<Editor />` and a separate view without all the editor stuff (like settings, toolboxes etc.) and render a `disabled` `<Editor />`.

That being that, this is not the same as suggested in #42. We still have a larger footprint because we need to load all the components inside the resolver (although we could analyze the JSON first and only load the needed components) and we could not do SSR. But... it works :-)
